server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/model/vote/IRVChoices.java

```diff
@@ -88,16 +88,24 @@ public int getRawChoicesCount() {
   /**
    * Constructor - takes a list of IRVPreferences (ranked candidates) representing a vote, sorts
    * this list by rank (most to least preferred), then stores in an unmodifiable list.
```

Comment:
Method comment does not mention firstDropped, what it is & what the method does with it.

Reply:
Thanks - actually I realised that that call was unnecessarily complicated (it was deleting the stuff off the end and then clearing it). I have now substituted a simpler sublist call.